package.json

```
@@ -100,10 +109,9 @@
     "lint-arg": "eslint --report-unused-disable-directives",
     "lint-fix": "npm run lint-arg -- --fix .",
     "prepare": "husky install",
-    "test": "cross-env BABEL_ENV=test nyc --reporter text-summary mocha --reporter dot --recursive --require @babel/register --timeout 12000",
-    "test-cov": "cross-env BABEL_ENV=test nyc mocha --reporter dot --recursive --require @babel/register --timeout 12000",
-    "test-index": "cross-env BABEL_ENV=test mocha --recursive --require @babel/register --reporter progress --timeout 12000 test/rules/index.js",
-    "test-no-cov": "cross-env BABEL_ENV=test mocha --reporter dot --recursive --require @babel/register --timeout 12000"
+    "test": "mocha",
+    "test-cov": "cross-env BABEL_ENV=test nyc npm run test",
```

The purpose of `test-cov` is to see a table of line numbers indicating failing coverage, so we should override the reporter in that case (otherwise, I kept `test` in case @gajus didn't want that large summary always showing).

```suggestion
    "test-cov": "cross-env BABEL_ENV=test nyc --reporter text npm run test",
```

FWIW, I switched from `progress` to `dot` for the Mocha reporter in anticipation of ESLint supporting parallel tests for Mocha (still hasn't happened, but should be coming) as `progress` doesn't work with parallel ones. We could use it for `test-index`, but I guess we may as well be uniform.

---

Should this be set in the config like the others, instead of the current `text-summary`?

---

I got the impression @gajus might like the minimalism of `text-summary` normally. But I'd like to have one script to get the full results.

---

OK, but with this change, it doesn't run coverage as the default anymore, only for this task, so it's already minimal as the default, if you wanted to have the verbose for this coverage task

---

I mean I think he prefers the minimal coverage reporter--just to see the summary that coverage is 100% or whatever.